### PR TITLE
Linksharing design updates

### PR DESCRIPTION
**Pull request by NikolaiYurchenko**

### CODE REVIEW

**Comment by Erikvv** [comment]
> i still need to add few more changes before your testing. 

Sorry I didn't notice the badge

> and for set-upping this locally, there is a doc for it in confluence

That's a bit confusing because I followed the readme and it pointed me to https://github.com/storj/gateway-mt/blob/main/docs/linksharing.md#development

**Comment by NikolaiYurchenko** [comment]
> > i still need to add few more changes before your testing.
> 
> Sorry I didn't notice the badge
> 
> > and for set-upping this locally, there is a doc for it in confluence
> 
> That's a bit confusing because I followed the readme and it pointed me to https://github.com/storj/gateway-mt/blob/main/docs/linksharing.md#development

try [this](https://storjlabs.atlassian.net/wiki/spaces/ENG/pages/1587740718/Running+linksharing+locally)

**Comment by Erikvv** [comment]
If you rebase the error should be gone.

I'm testing but I keep getting 404 in the auth service. I'm trying to find out what's going wrong.

```
$ uplink --access=12edqrJ1APojgZRKmvQCS8uGVc55t8SXf34Y79e1JfTv3M4cmpfAJ9HikGi3R5wBzPvDWQCxskBoS74fowYMtBkp8PEWYAAh7J9bLaPtz3msML6q8Q6P1cGrsuWTWTNrUgdb6wFY95J4pQFcZwoNMAsqaWsUvEtkHd5CyeCZe1KppRttUTwZRqHu68sDrqFCfxST3ytLEokzuGp2CR37r5c1Grm8yHqHKy2ZPiyNrJ4uuBZSiShQX9mRfqw share --url --auth-service=http://localhost:8000 --base-url=http://localhost:8001 sj://testbucket2/cert.pem
Sharing access to satellite 12AACAKx9MQcNxNurPnKUrTZ2qhqX97dRbFSPePcuVjY5grkN9C@storjsim:10000
=========== ACCESS RESTRICTIONS ==========================================================
Download  : Allowed
Upload    : Disallowed
Lists     : Allowed
Deletes   : Disallowed
NotBefore : No restriction
NotAfter  : No restriction
Paths     : sj://testbucket2/cert.pem
=========== SERIALIZED ACCESS WITH THE ABOVE RESTRICTIONS TO SHARE WITH OTHERS ===========
Access    : 13Rkgm2MWwEyU53vVy4YuQpxaKMMwdPSd9pEeq1GBDwpjBPsTb3uaKZPDYNPPxxy1vC6DiDZZteU3bc6HA967xe1qA2SoUceBQkQg5eJgj3QYqvf7XAP4HzuCm5aw5gkQYDUjDLNnUMUFHGj9uKn4jgyaJZeUZgrk4JAU2AJWKpu7pBJoHFMVoJq2UiCnQsBHirQoX2P73q5KuVvrqwavP98FV4EikBmrbpcXmGv6L9x1YDHrWJowLufyaKdUmRh5KdMWaJNpfJtnmeCo79i4pERW8qrR9jvKR6GiqJAugq5djgx5BjGXjeMnYfRgA2A1bCgPcFQWzrgQGmdSVFti1jUC8TM2RMq6ZN17wmX77LfNKBqJBa5NsUkqUDqLYExdVYpQ4smDXUa4Kbeez8Tz5vBPYjrj982nt1h3ARNgXovAX5FbJqnqS
========== CREDENTIALS ===================================================================
Access Key ID: jx6xs36zier3hrfbbnz73phxuymq
Secret Key   : j27cuxew3a4qrsksvlqfbgks6zrmitgty3oh3lyuoms3m6w54pjpy
Endpoint     : http://localhost:7777
Public Access:  true
=========== BROWSER URL ==================================================================
REMINDER  : Object key must end in '/' when trying to share recursively
URL       : http://localhost:8001/s/jx6xs36zier3hrfbbnz73phxuymq/testbucket2/cert.pem
```

```
linksharing_1  | 2021-08-31T17:32:19.302Z	INFO	httpserver/log.go:17	access	{"method": "GET", "host": "localhost:8001", "user-agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36"}
linksharing_1  | 2021-08-31T17:32:19.302Z	DEBUG	sharing/handler.go:255	unable to handle request	{"error": "auth service: invalid status code: 404", "action": "unknown", "status_code": 404}
linksharing_1  | 2021-08-31T17:32:19.302Z	INFO	httpserver/log.go:54	response	{"method": "GET", "host": "localhost:8001", "code": 404, "user-agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36", "content-length": 0, "written": 4181, "duration": "146.987µs"}
```

```
authservice_1  | 2021-08-31T17:25:26.068Z	INFO	server/log.go:36	access	{"method": "POST", "host": "localhost:8000", "user-agent": "Go-http-client/1.1"}
authservice_1  | 2021-08-31T17:25:26.068Z	DEBUG	resources	httpauth/resources.go:142	newAccess request	{"remote address": "172.20.0.1:37880"}
authservice_1  | 2021-08-31T17:25:26.068Z	INFO	server/log.go:170	response	{"method": "POST", "host": "localhost:8000", "code": 200, "user-agent": "Go-http-client/1.1", "content-length": 471, "written": 153, "duration": "140.953µs"}
authservice_1  | 2021-08-31T17:25:32.821Z	INFO	server/log.go:36	access	{"method": "GET", "host": "authservice:8000", "user-agent": "Go-http-client/1.1"}
authservice_1  | 2021-08-31T17:25:32.821Z	INFO	server/log.go:170	response	{"method": "GET", "host": "authservice:8000", "code": 404, "user-agent": "Go-http-client/1.1", "content-length": 0, "written": 0, "duration": "59.055µs"}
```

**Comment by Erikvv** [comment]
If I compare the live version with this one the download button is now below the fold on smartphone.

live:
prod

This one:
test

**Comment by Erikvv** [comment]
Also in production for small files there is the message "files under 4k are stored as metadata with strong encryption" instead of "0 nodes are storing a piece of this file"

File: pkg/linksharing/web/static/img/collapse-left.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
+	 viewBox="0 0 512 512" style="enable-background:new 0 0 512 512;" xml:space="preserve">
+	<g><path d="M0,0v512h64V0H0z M384,0v512h128V0H384z M320,128L192,256l128,128V128z"/></g>
+</svg>
```
Comment (Erikvv) [inline pkg/linksharing/web/static/img/collapse-left.svg:4]: This image has black bars on the side. 

Also the sideways arrow only makes sense on a large viewport.

**Comment by Erikvv** [comment]
Is there a mockup or set of requirements for this?

**Comment by Erikvv** [comment]
Error page looks better, nice.

**Comment by NikolaiYurchenko** [comment]
> Error page looks better, nice.

design: https://www.figma.com/file/oPwnXJym3nrN05nXoSiOAm/Storj.io?node-id=2741%3A6835
collapse icon is temporary because of no design for it and mobile adaptation is only for not breaking things)
About 0 locations, should we hile if 0?

**Comment by iglesiasbrandon** [comment]
@NikolaiYurchenko @Erikvv we could hide the message that says "0 Storj nodes are storing a piece of this file" but I think it might be better is for us to get this version out into production with the current message and then create some updated designs for the inline file edge case. A better experience inline files would be to not even show a map and have some copy around why the file is stored on the satellite or something like that.

**Review by Erikvv — DISMISSED** [review]
Is a bit of a UX regression on mobile but it does work and the call to action (sign up) is there.

File: pkg/linksharing/web/single-object.html [hunk 2]
```diff
@@ -146,8 +129,38 @@ <h5 class="modal-title mx-auto" id="shareModalLabel">Share {{.Data.Key}}</h5>
       }
   }
 
+  function toggleSideBar() {
+      const sidebar = document.querySelector('.sidebar');
+      const contentArea = document.querySelector('.content')
+      const expander = document.querySelector('.expander');
+
+      if (sidebar && contentArea && expander) {
```
Comment (crawter) [inline pkg/linksharing/web/single-object.html:137]: if (!(sidebar && contentArea && expander)) {
    return;
}

...

Comment (crawter) [reply pkg/linksharing/web/single-object.html:137]: small if always easier to read

**Review by crawter — APPROVED** [review]

**Review by VitaliiShpital — APPROVED** [review]

**Pull request merged**